addins/Cake.Board.AzureBoards.yml

```diff
@@ -2,8 +2,10 @@ Name: Cake.Board.AzureBoards
 NuGet: Cake.Board.AzureBoards
 Assemblies:
 - "/**/Cake.Board.AzureBoards.dll"
+- "/**/Cake.Board.Abstractions.dll"
+- "/**/Cake.Board.dll"
 Repository: https://github.com/nicolabiancolini/Cake.Board
-Author: nicolabiancolini
```

Comment:
@jericho will this change cause problems with the auto updater?

Reply:
The AddinDiscoverer will submit a PR to change this back because we favor the repo owner name when generating the YAML.